src/main/res/values-fr/string.xml

```diff
@@ -126,7 +126,7 @@
 
     <!-- Main Menu -->
 
-    <string name="mainmenu_frontpage">Front Page</string>
+    <string name="mainmenu_frontpage">Page d'Accueil</string>
```

Comment:
I think the build is failing, because this apostrophe needs to be escaped with a backslash:

```suggestion
    <string name="mainmenu_frontpage">Page d\'Accueil</string>
```